buttons-panel-modal-example/src/app/extensions/components/ModalExample.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  ModalBody,
+  ModalFooter,
+  Modal,
+  Heading,
+  Button,
+  Input,
+  Flex,
+  Form,
+  Tag,
+  Text,
+} from '@hubspot/ui-extensions';
+
+export const ModalExample = ({ actions }) => {
+  return (
+    <Modal id="default-modal" title="Dialog Header" width="md">
+      <ModalBody>
+        <Flex direction="column" gap="small">
+          <Text>
+            When we get all the 0's and 1's line up, a link will show in
+            your&nbsp;
```

Comment:
nit: unintentional &nbsp;? I haven't seen one of those in a minute! 
```suggestion
            your
```

Reply:
This was intentional bc adding a manual space wasn't rendering properly with the next `<Text>` block